lib/src/components/Banner.tsx

```diff
@@ -58,33 +122,31 @@ function	Banner({
 	}
 	return (
 		<div
-			className={`transition-max-height overflow-hidden duration-600 ${isVisible ? maxHeight : 'max-h-0'}`}>
-			<div className={`Banner--wrapper flex relative rounded-lg border-2 overflow-hidden ${bannerClassName} ${variant === 'image' && "bg-no-repeat bg-cover bg-center bg-[url('https://s3-alpha-sig.figma.com/img/51a1/2fe5/7e3ce66410a263a2c114465d9983e44c?Expires=1652054400&Signature=M75j71LEDL-7A95sBpXcuAXLZ~H06v0GyWZgZEfuA~-aX4Ouc3V6brvl-B0-WL5rU8-mDRGaIk2TnYx-FnZN-NYg5vMCT1FT~ehpUA~XN5emO~zPY~7N-AJbhIPxEX9OI137ysqsQs72~RuBPoNfyJRaGY92SFPBV~ity3xcI-~evMzP3h3UDUU~VctTGlbDIl7wSXOr~S1PTesgClzh-9nVBxyVrTB2eUW~-L6wK765CwfLJHjw8rWjKO8D336O9dJ0mU7vQShq2rr6tgS0EA~evYLMf-3JWl5cBKea6GGALcqebJIlGC6PxkSwQ27SVe0mtr8zZXX8~WwOpdRzvg__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA')]"}`}>
-				<div className={`p-6 flex-col ${variant === 'split' && 'w-1/2'}`}>
-					{canClose ? (
-						<button onClick={onTryToClose} className={'absolute top-4 right-4'}>
-							<IconCross className={'w-6 h-6 cursor-pointer'} />
-						</button>
-					) : null}
-					{variant === 'image'
-						? <pre className={'mb-6 text-inherit text-4xl'}>{title}</pre>
-						: <h4 className={'mb-6 text-inherit'}>{title}</h4>
-					}
+			className={'transition-max-height overflow-hidden duration-600'}
+			style={{maxHeight: isVisible ? contentHeight : 0}}>
```

Comment:
MaxHeight as style instead of className because we are setting it with Javascript. This is way easier this way